AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -166,17 +166,25 @@ public async Task<IActionResult> PatchAssetAddress([FromRoute] EditAssetByIdRequ
         {
             var header = HttpContext.Request.Headers.GetHeaderValue(HeaderConstants.IfMatch);
 
+            int numericValue;
+
             if (header == null)
                 return null;
 
+            if (header.GetType() == typeof(string))
+            {
+                if (int.TryParse(header, out numericValue))
+                    return numericValue;
+            }
+
             _ = EntityTagHeaderValue.TryParse(header, out var entityTagHeaderValue);
```

Comment:
Have you tested this by running it locally?

Reply:
Yes, if you send the if-match as a string, it comes through and it does NOT hit line 174, as the type is actually not string (I think it's an object), however if you send the if-match as a number (no quotes), it comes through as a string, hits line 174, and the correct numeric value, type int, is returned.